libraries/pom.xml

```diff
@@ -8,13 +8,13 @@
     </parent>
 
     <artifactId>libraries</artifactId>
+    <version>2.0.2-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <name>(Comcast) ${project.artifactId}</name>
     <description>DAWG library</description>
 
     <modules>
-        <module>dawg-config</module>
```

Comment:
I removed this... Seems dawg-config is literally just an example config file. We can keep it in the repo for reference but I am not so sure it needs get put into an RPM and be part of the build and deployment.